pom.xml

```diff
@@ -322,7 +320,7 @@
           <version>${version.plugin.assembly}</version>
           <configuration>
             <archive>
-              <index>true</index>
+              <!-- Don't generate index as causes https://github.com/gradle/gradle/issues/15269 -->
```

Comment:
I am a bit worried about this change since I think it affects anything that uses this POM as a parent (e.g., build-finder, RWX, kojiji), so I am not sure why a problem if it is specific to PME should propagate to other projects.

Reply:
Theoretically its not a problem _just_ for PME but for any project building a jar which _could_ be used by another project that uses **Gradle**. I'm happy to discuss this - the idea was to transparently fix it for everyone rather than more projects suffering the same problems that GME did (until the index marker inside PME jars were discovered)